Cargo.toml

```diff
@@ -60,7 +60,7 @@ hyper-timeout = "0.5.1"
 hyper-util = "0.1.11"
 json-patch = "4"
 jsonpath-rust = "0.7.3"
-k8s-openapi = { version = "0.25.0", default-features = false }
+k8s-openapi = { git = "https://github.com/Arnavion/k8s-openapi.git", branch = "master", default-features = false }
```

Comment:
Given that we are doing a temporary git PIN here, we should put a rev in.

We have to fix this before releasing, but that's fine. At least we can get our `main` into a good state and keep iterating until k8s-openapi makes a new release.